package.json

```diff
@@ -46,21 +50,27 @@
     "postcss-discard-duplicates": "^2.0.2",
     "postcss-import": "^9.1.0",
     "postcss-loader": "^1.2.2",
-    "standard": "^8.6.0",
+    "standard": "^10.0.2",
     "style-loader": "^0.13.1",
     "stylus": "^0.54.5",
     "stylus-loader": "^3.0.1",
     "svg-sprite-loader": "^0.2.0",
     "url-loader": "^0.5.7",
     "webpack": "^1.14.0",
-    "webpack-dev-server": "^1.16.2",
     "webpack-merge": "^2.4.0",
     "yaml-loader": "^0.4.0"
   },
   "dependencies": {
+    "date-fns": "^1.28.5",
     "node-polyglot": "^2.2.2",
     "piwik-react-router": "^0.8.2",
     "preact": "^8.1.0",
     "preact-compat": "^3.16.0"
+  },
+  "standard": {
+    "env": [
+      "browser",
+      "jest"
```

Comment:
Maybe later we will add another test runner. Maybe later we will not remember why there is such a configuration in this very file. The butterfly effect and the chaos theory warn us not to expect what the future will be and we have enough to do to manage present and needed things.

But it is a detail, and noone cares for such little not annoying things but me, too perfectionist. Sorry if it bothers you.